force-app/main/default/objects/Review__c/fields/SubmittedDate__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>SubmittedDate__c</fullName>
```

Comment:
This is the new field "Submitted Date" - I added some basic help text and a description as well. Let me know what you think @nicolersmith

Reply:
@nicolersmith , also note in docs that the `SubmitReview` Quick Action is packaged that means:
- The layout and pre-defined fields is editable by customers
- But the Quick Action itself (see screenshot) is not editable.
    - @ClayTomerlin , is it possible yet to use a Custom Label for the Quick Action **Success Message** since customers will not be able to change this value?   Or is it something that can be translated in the Translation Workbench?

![SubmitReivew QuickAction](https://user-images.githubusercontent.com/46537641/116899583-ba93e280-ac05-11eb-9a76-bcf74087ddca.png)

Reply:
Here's a screenshot of translating the `SubmiteReivew` Quick Action.   I see the label is translatable but not the **Success Message** :cry: 

![Translate SubmitReview Quick Action](https://user-images.githubusercontent.com/46537641/116901451-f0d26180-ac07-11eb-9468-5cb25a23489a.png)

Reply:
Ok, [translating the Success Message is possible](https://help.salesforce.com/articleView?id=sf.actions_custom_success_messages.htm&type=5).   It has the `Informational Message` **Aspect**.
![Translate Success Message](https://user-images.githubusercontent.com/46537641/116907241-54ac5880-ac0f-11eb-8e8f-8b9c1b92e09f.png)

Thank you!

Reply:
@scottwarren-sfdo Thank you for asking this question and also doing the research, this is super important. It doesnt look like any changes need to be made to help with this but let me know.